docs/versioning.md

```diff
@@ -0,0 +1,153 @@
+# Electron Versioning
+
+> This document is a detailed look at our versioning policy and implementation.
+
+For most people, the following will provide you with the most recent stable build:
+
+```bash
+$ npm install --save electron
+```
+
+If you want to track the latest electron _always_,
+you will need to run `npm install --save electron@latest` whenever a new major version is released.
+
+## Version 1.x
+
+Electron versions *< 2.0* have not conformed to the [semver](http://semver.org) spec. Major versions corresponded to end-user API changes. Minor versions corresponded to Chromium major releases. Patch versions corresponded to new features and bug fixes. While convenient for developers merging features, it creates problems for developers of client-facing applications. The QA testing cycles of major apps like Slack, Stride, Teams, Skype, VS Code, Atom, and Desktop can be lengthy and stability is a highly desired outcome. There is a high risk in adopting new features while trying to absorb bug fixes.
+
+Here is an example of the 1.x strategy:
+
+![](versioning/sketch-0.png)
```

Comment:
Please move all images to `/docs/images` and update all image URLs from relative to fully-qualified by prepending `https://cdn.rawgit.com/electron/electron/master/docs/images/`

See https://github.com/electron/electron/pull/11314 for context on this.